libs/ledger-live-common/src/__tests__/test-helpers/bridge.ts

```diff
@@ -131,6 +131,7 @@ export function testBridge<T extends TransactionCommon>(data: DatasetTest<T>): v
         return implicitMigration(accounts);
       } catch (e: any) {
         console.error(e.message);
+        releaseMockDevice(deviceId);
```

Comment:
So `finally` isn't doing what we think it does? :open_mouth:

Reply:
Nah sorry i removed it, i didn't understand a specific bug and i was trying a lot of things :laughing: